ReactAndroid/src/main/java/com/facebook/react/ReactPerformanceLogger.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
+ * directory of this source tree.
+ */
+package com.facebook.react;
+
+import android.util.Log;
+
+import com.facebook.react.bridge.ReactMarker;
+import com.facebook.react.bridge.ReactMarkerConstants;
+import com.facebook.react.common.annotations.VisibleForTesting;
+
+import javax.annotation.Nullable;
+
+import static com.facebook.react.bridge.ReactMarkerConstants.RUN_JS_BUNDLE_END;
+
+public class ReactPerformanceLogger implements ReactMarker.MarkerListener {
+
+  public enum Tag {
+    BRIDGE_STARTUP,
+    SCRIPT_EXECUTION,
+  }
+
+  private static final String TAG = ReactPerformanceLogger.class.getSimpleName();
+
+  private long[][] mData = new long[Tag.values().length][2];
+
+  public ReactPerformanceLogger() {
+    ReactMarker.addListener(this);
+  }
+
+  public long getDuration(Tag tag) {
+    return mData[tag.ordinal()][1] - mData[tag.ordinal()][0];
+  }
+
+  public void release() {
+    ReactMarker.removeListener(this);
+  }
+
+  @Override
+  public void logMarker(ReactMarkerConstants name, @Nullable String tag, int instanceKey) {
+    switch (name) {
+      case GET_REACT_INSTANCE_MANAGER_START:
```

Comment:
Why are we only marking these ? Why not just pick up all markers that RN gives ?